grammars/asciidoc.cson

```diff
@@ -64,17 +80,123 @@
     'begin': '(^/{4,}$)'
     'end': '\\1'
     'name': 'comment.block.asciidoc'
+    'patterns': [
+      { 'include': '#inline' }
+    ]
+  }
+
+  # Matches example block
+  #
+  # Examples
+  #
+  #   ====
+  #   A multi-line example.
+  #
+  #   Notice it's a delimited block.
+  #   ====
+  {
+    'begin': '^(={4,})+?$'
```

Comment:
I've noticed this pattern differ from other block patterns. Why do you need `+?` after the group?